openmc/mgxs_library.py

```diff
@@ -1638,9 +1638,9 @@ def set_multiplicity_matrix_mgxs(self, nuscatter, scatter=None,
                                    xs_type=xs_type, moment=0,
                                    subdomains=subdomain)
             if scatter.scatter_format == 'histogram':
-                scatt = np.sum(scatt, axis=0)
+                scatt = np.sum(scatt, axis=2)
```

Comment:
Whew. good find @GiudGiud! I was sitting here wondering how I never ran in to this before and I guess it purely has been because I have always used an openmc.mgxs.MultiplicityMatrixXS object. This definitely speaks to the value of independent users!!